providers/implementations/macs/kmac_prov.c

```diff
@@ -686,6 +689,9 @@ static void *kmac##size##_internal_new(void *provctx)                          \
 {                                                                              \
     struct kmac_data_st *macctx = kmac##size##_new(provctx);                   \
                                                                                \
+    if (!ossl_prov_is_running())                                               \
```

Comment:
This one isn't required because of the check in `kmac_new()` which gets called initialising `macctx`.